index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Intuit Accounts - Signup</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <script src="https://kit.fontawesome.com/f977934f89.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+    <header class="header">
+
+        <div class="signup_header">
+            
+            <div>
+                <p class="already_have_account">Already have an Intuit account?</p>
+                <a href="#" class="forget_password">I forgot my User ID or Password</a>
+            </div>
+
+            <div>
+                <button class="sign_in fas fa-lock"><span class="btn_text">Sign In</span></button>
+            </div>
+
+        </div>
+
+    </header>
+
+    <article class="company-info">
+        <h6>
+            <img src="./images/intuit_logo.png" class="logo" alt="Intuit Logo">
+        </h6>
+        <div class="info_icons">
+            <div class="icon_info">
+                <img class="icon" src="./images/mint.svg" alt="Mint Icon">
+                <span>mint</span>
+            </div>
+            <div class="icon_info">
+                <img class="icon" src="./images/quickbooks.svg" alt="Quickbooks Icon">
+            <span>quickbooks</span>
+            </div>
+            <div class="icon_info">
+                <img class="icon" src="./images/turbotax.svg" alt="Turbotax Icon">
+                <span>turbotax</span>
+            </div> 
```

Comment:
- [ ] Kindly check the size of each text.

> Original
![image](https://user-images.githubusercontent.com/57414789/82000378-c9147300-961d-11ea-9ade-50a3c2b2ce10.png)

> Your page
![image](https://user-images.githubusercontent.com/57414789/82000392-d92c5280-961d-11ea-8e35-0e058049cceb.png)